server/common/app_config.py

```diff
@@ -414,8 +432,12 @@ def __init__(self, app_config, default_config):
         # The matrix data cache manager is created during the complete_config and stored here.
         self.matrix_data_cache_manager = None
 
+        # The authentication object (BCM -- better name)
```

Comment:
could you write this as ```# TODO @brian better name```

Reply:
Oops, didn't mean to commit the "BCM".   I'll fix this and the other issue in my next PR.